mendix/widgets-resources
# Data grid - Introduce row concept

**What I am doing in this PR?**
In this PR I am introducing a pseudo-row concept to the datagrid, which allows customers to define their own hover styles or styles based on rows.

**Why?**
Because in the current DOM we cant style a row as a row due to CSS GRID. It does create automatically the rows based on the amount of columns defined in `grid-template-columns` property.

**How to test?**
The behavior should remain the same, it means while reordering, or resizing. If adding a style with the following class it should apply hover styles to the entire row. PS: `Dynamic row class` should also be tested.
```
.tr:hover > .td {
    background-color: red;
}
```
